docs/reference/ml/apis/put-dfanalytics.asciidoc

```diff
@@ -57,23 +57,22 @@ and mappings.
 [[ml-put-dfanalytics-request-body]]
 ==== {api-request-body-title}
 
-`source` (Required)::
-  (object) The source configuration, consisting of `index` and optionally a 
-  `query`.
-  
-`dest` (Required)::
-  (object) The destination configuration, consisting of `index` and optionally 
-  `results_field` (`ml` by default).
-  
 `analysis` (Required)::
   (object) Defines the type of {dfanalytics} you want to perform on your source 
-  index. For example: `outlier_detection`. 
-  See {oldetection} resources.
+  index. For example: `outlier_detection`. See <<dfanalytics-types>>.
   
 `analyzed_fields` (Optional)::
   (object) You can specify both `includes` and/or `excludes` patterns. If 
-  `analyzed_fields` is not set, only the relevant fileds will be included. For 
-  example all the numeric fields for {oldetection}.
+  `analyzed_fields` is not set, only the relevant fields will be included. For 
+  example, all the numeric fields for {oldetection}.
+  
+`dest` (Required)::
```

Comment:
I sorted these properties in the create DF analytics API and added links to the appropriate sections in the resource page.